freqtrade/rpc/rpc.py

```diff
@@ -226,6 +226,53 @@ def _rpc_daily_profit(
             for key, value in profit_days.items()
         ]
 
+    def _rpc_trade_history(
+            self, last_trades_number: int) -> Dict[str, List[Dict[str, Any]]]:
+        """ Returns the X last trades """
+        if last_trades_number > 0:
```

Comment:
i somehow don't like the wording of `last_trades_number` ... but i currently struggle to really find a better name - maybe something as simple as `limit` ?

Reply:
yes limit is good.